scripts/release_ios_sdk.sh

```
@@ -1,305 +1,533 @@
+  generate_build_token_file "$build_token" "$worktree_token_path"
+
+  # Commit BuildToken.swift in worktree
+  pushd "$worktree_dir" >/dev/null
+  git add -f "$TOKEN_ABS_PATH"
+  git commit -m "Add BuildToken.swift for release v$new_version
+
+SECURITY: BuildToken.swift is in .gitignore and NOT in main branch.
+This file is ONLY included in release tags for SPM distribution.
+
+Token: $build_token
+Generated: $(date -u +"%Y-%m-%d %H:%M:%S UTC")
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)